Update all docker container references to point to GitHub container registry (#1013)
informalsystems/apalache: docs/src/apalache/installation/docker.md

To get the latest Apalache image, issue the command:

```bash
docker pull ghcr.io/informalsystems/apalache
```

## Running the docker image

To run an Apalache image, issue the command:

```bash

> $ docker run --rm -v <your-spec-directory>:/var/apalache informalsystems/apalache <args>

Reviewer: Just tried it. You have to use `ghrc.io/informalsystems/apalache`
```suggestion
$ docker run --rm -v <your-spec-directory>:/var/apalache ghrc.io/informalsystems/apalache <args>
```

  When using SELinux, you might have to use the modified form of `-v` option:
    `-v <your-spec-directory>:/var/apalache:z`

> - `informalsystems/apalache` is the APALACHE docker image name. By default, the `latest` stable

Reviewer: ```suggestion
- `ghrc.io/informalsystems/apalache` is the APALACHE docker image name. By default, the `latest` stable
```

>   version is used; you can also refer to a specific tool version, e.g., `informalsystems/apalache:0.6.0` or `informalsystems/apalache:unstable`

Reviewer: ```suggestion
  version is used; you can also refer to a specific tool version, e.g., `ghrc.io/informalsystems/apalache:0.16.0` or `ghrc.io/informalsystems/apalache:unstable`
```

###### using the latest stable

$ alias apalache='docker run --rm -v $(pwd):/var/apalache informalsystems/apalache'

###### using the latest unstable

> $ alias apalache='docker run --rm -v $(pwd):/var/apalache informalsystems/apalache:unstable'

Reviewer: ```suggestion
$ alias apalache='docker run --rm -v $(pwd):/var/apalache ghrc.io/informalsystems/apalache:unstable'
```

on the unstable branch for the description of the newest features. **We
recommend using the unstable version if you want to try all the exciting new
features of Apalache. But be warned: It is called "unstable" for a reason**. To

> use `unstable`, just type `informalsystems/apalache:unstable` instead of `informalsystems/apalache`

Reviewer: ```suggestion
use `unstable`, just type `ghrc.io/informalsystems/apalache:unstable` instead of `ghrc.io/informalsystems/apalache`
```

everywhere.

Do not forget to pull the docker image from time to time:

```bash
docker pull ghcr.io/informalsystems/apalache:unstable
```

Run it with the following command:

```bash

> $ docker run --rm -v <your-spec-directory>:/var/apalache informalsystems/apalache:unstable <args>

Reviewer: ```suggestion
$ docker run --rm -v <your-spec-directory>:/var/apalache ghrc.io/informalsystems/apalache:unstable <args>
```

```

To create an alias pointing to the `unstable` version:

```bash

> $ alias apalache='docker run --rm -v $(pwd):/var/apalache informalsystems/apalache:unstable'

Reviewer: ```suggestion
$ alias apalache='docker run --rm -v $(pwd):/var/apalache ghrc.io/informalsystems/apalache:unstable'
```
